Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -620,23 +634,36 @@ extension AlertManager: AlertPermissionsCheckerDelegate {
         } else {
             if alreadyIssued {
                 setAlreadyIssued(false)
-                retractAlert(identifier: alert.identifier)
+                retractionHandler(alert)
             }
             return false
         }
     }
-}
 
-fileprivate extension AlertManager {
-    private var isAppInBackground: Bool {
-        return UIApplication.shared.applicationState == UIApplication.State.background
+    private func presentUnsafeNotificationPermissionsInAppAlert() {
+        DispatchQueue.main.async {
+            let alertController = AlertPermissionsChecker.constructUnsafeNotificationPermissionsInAppAlert() { [weak self] in
+                self?.acknowledgeAlert(identifier: AlertPermissionsChecker.unsafeNotificationPermissionsAlertIdentifier)
+            }
+            self.alertPresenter.present(alertController, animated: true) { [weak self] in
+                // the completion is called after the alert is presented
+                self?.unsafeNotificationPermissionsAlertController = alertController
+            }
+        }
+    }
+
+    private func dismissUnsafeNotificationPermissionsInAppAlert() {
+        guard let alertController = unsafeNotificationPermissionsAlertController else { return }
+        alertPresenter.dismissAlert(alertController, animated: true) { [weak self] in
+            self?.unsafeNotificationPermissionsAlertController = nil
+        }
     }
 }
 
 fileprivate extension UserDefaults {
     private enum Key: String {
-        case hasIssuedNotificationPermissionsAlert = "com.loopkit.Loop.HasIssuedNotificationPermissionsAlert"
-        case hasIssuedScheduledDeliveryEnabledAlert = "com.loopkit.Loop.HasIssuedScheduledDeliveryEnabledAlert"
+        case hasIssuedNotificationPermissionsAlert = "com.loop.Loop.HasIssuedNotificationPermissionsAlert"
```

Comment:
Guessing this was an accidental find/replace?

Reply:
I'll change back to loopkit